packages/fxa-content-server/app/scripts/views/pair/auth_complete.js

```diff
@@ -2,30 +2,78 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import BaseView from '../base';
 import Cocktail from 'cocktail';
 import DeviceBeingPairedMixin from './device-being-paired-mixin';
 import PairingGraphicsMixin from '../mixins/pairing-graphics-mixin';
 import Template from '../../templates/pair/auth_complete.mustache';
+import { assign } from 'underscore';
+import preventDefaultThen from '../decorators/prevent_default_then';
+import WebChannel from '../../lib/channels/web';
+import Constants from '../../lib/constants';
+import FormView from '../form';
 
-class PairAuthCompleteView extends BaseView {
+class PairAuthCompleteView extends FormView {
   template = Template;
 
+  events = assign(this.events, {
+    'click #open-firefox-view': preventDefaultThen('openFirefoxView'),
+  });
+
+  initialize(options) {
+    super.initialize(options);
+    this._notificationChannel = this.createWebchannel();
+  }
+  
+  createWebchannel() {
+    // This is yuck, we should probably pull this into a specific relier class
```

Comment:
Maybe file a maintenance bug for this as well? If we used the BaseView, would we have been able to use the default web channel?